jekyll/_cci2/artifacts.md

```diff
@@ -39,10 +41,37 @@ jobs:
           path: /tmp/artifacts
 ```
 
-Using the `store_artifacts` step, we upload 2 build artifacts: a file (`/tmp/artifact-1`) and a directory (`/tmp/artifacts`). Once the artifacts are successfully uploaded, you can view them in the **Artifacts** tab of the build page in your browser. There is no limit on the  number of `store_artifacts` steps a job can have.
+The `store_artifacts` step uploads two build artifacts: a file (`/tmp/artifact-1`) and a directory (`/tmp/artifacts`). After  the artifacts successfully upload, view them in the **Artifacts** tab of the build page in your browser. There is no limit on the number of `store_artifacts` steps a job can run.
 
-Currently, `store_artifacts` takes 2 fields: `path` and `destination`.
+Currently, `store_artifacts` has two keys: `path` and `destination`.
 
   - `path` is a path to the file or directory to be uploaded as artifacts.
-  - `destination` **(Optional)** is a prefix added to the artifact paths in the artifacts API. (default: the directory of the file specified in `path`)
+  - `destination` **(Optional)** is a prefix added to the artifact paths in the artifacts API. The directory of the file specified in `path` is used as the deafult.
+  
+## Download All Artifacts for a Build on CircleCI  
+
+Use the following procedure to download your artifacts with `curl`.
+
+1. Create an API token in the application by going to [User Settings > API Tokens](https://circleci.com/account/api).
+
+2. Click Create New Token, add a name in the dialog box and click Add API Token.
```

Comment:
I would add something here to tell the user what the name is for, so that they are not wondering what the name should be. Maybe something like  "The name of the token is to help your keep track of the tokens that you have create"